indico/modules/events/papers/forms.py

```diff
@@ -49,3 +51,21 @@ def __init__(self, *args, **kwargs):
             del self.content_reviewers
         if not self.event.cfp.layout_reviewing_enabled:
             del self.layout_reviewers
+
+
+class PaperReviewingSettingsForm(IndicoForm):
+    content_review_questions = PaperReviewQuestionsField(_("Content review questions"))
+    layout_review_questions = PaperReviewQuestionsField(_("Layout review questions"))
+    announcement = IndicoMarkdownField(_('Announcement'), editor=True)
+
+    @generated_data
+    def announcement_render_mode(self):
```

Comment:
Then we can get rid of the render_mode and always use `MarkdownText(..)` in the `announcement` property in the `CallForPapers`